_includes/navigation.html

```diff
@@ -4,7 +4,7 @@ <h1 id="apex-title">APEX</h1>
     <h2 id="dpg">DIABOLO PERFORMANCE GROUP</h2>
   </a>
   <h3 class="nav-link">
-    <a class="no-decoration" href="https://youtu.be/@apex.diabolo" target="blank">
+    <a class="no-decoration" href="https://youtube.com/@apex.diabolo" target="blank">
```

Comment:
Did the shortlink not work for you? YouTube uses shortlinks for its own link sharing, maybe for cross-platform support?

Reply:
Yeah it would try to redirect to `https://www.youtube.com/watch?v=%40apex.diabolo&feature=youtu.be` and then that would redirect to `https://www.youtube.com/watch?v=%40apex.diabo` which is just not a valid video URL

Reply:
Oh shoot yeah, I had a third party redirector but you're right it doesn't work. Change looks good 👍